test/scripts/openshift-ci/setup-e2e-tests.sh

```diff
@@ -95,48 +124,88 @@ if [[ "$1" =~ raw ]]; then
   fi
 fi
 
-# Install KServe stack - skip serverless for raw, graph and predictor deployments
-if ! skip_serverless "$1"; then
+# Install KServe stack - skip serverless for raw, graph, predictor, and path_based_routing deployments
+# Also skip if using ODH operator (it installs them)
+if ! skip_serverless "$1" && [[ "$INSTALL_ODH_OPERATOR" == "false" ]]; then
   echo "Installing OSSM"
   $SCRIPT_DIR/deploy.ossm.sh
   echo "Installing Serverless"
   $SCRIPT_DIR/deploy.serverless.sh
 fi
 
-echo "⏳ Waiting for KServe CRDs"
-kustomize build $PROJECT_ROOT/config/crd | oc apply --server-side=true -f -
+# Install KServe components based on method
+if [[ "$INSTALL_ODH_OPERATOR" == "false" ]]; then
+  # Manual installation: Install KServe directly with PR images
+  echo "⏳ Installing KServe manually with PR images"
+  echo "⏳ Waiting for KServe CRDs"
+  kustomize build $PROJECT_ROOT/config/crd | oc apply --server-side=true -f -
+
+  wait_for_crd llminferenceserviceconfigs.serving.kserve.io 90s
+
+  echo "Installing KServe with Minio"
+  kustomize build $PROJECT_ROOT/config/overlays/test |
+    sed "s|kserve/storage-initializer:latest|${STORAGE_INITIALIZER_IMAGE}|" |
```

Comment:
These images also need to be updated if using the odh-operator; how is it handled?